### PR TITLE
HP-23: Create new fund

### PR DESCRIPTION
**Pull request by Nuta88**
image

### CODE REVIEW

File: src/app/modules/funds/funds.service.ts [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+import {Fund} from '../../shared/models/fund';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class FundsService {
-  constructor( private http: HttpClient ) {
-  }
+  private subject = new Subject<any>();
```
Comment (AlexKruhlov) [inline src/app/modules/funds/funds.service.ts]: Why do you use `any` type? Would it be better to have `Fund` instead `any` here?

File: src/app/modules/funds/funds.service.ts [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
 
-  getFunds(): Observable<Fund[]> {
-    return this.http.get<Fund[]>(`${URL}/funds`);
+  constructor(private http: HttpClient) {
   }
+
+  getFunds = (): Observable<Fund[]> => this.http.get<Fund[]>(`${URL}/funds`);
+
+  create = (fund: Fund): void => {
+    this.http.post(`${URL}/funds/create`, fund)
```
Comment (AlexKruhlov) [inline src/app/modules/funds/funds.service.ts]: It would be better to allocate `${URL}/funds` into separate constant and use it in all needed places.

**Review by AlexKruhlov — APPROVED** [review]

File: src/app/modules/funds/funds.service.ts (hunk 1 above)
Comment (Nuta88) [reply src/app/modules/funds/funds.service.ts]: done

File: src/app/modules/funds/funds.service.ts (hunk 2 above)
Comment (Nuta88) [reply src/app/modules/funds/funds.service.ts]: done

**Pull request merged**